mint/git-clone/mint-leaf.yml

```diff
@@ -32,14 +32,14 @@ tasks:
     run: |
       sudo install $MINT_LEAF_PATH/bin/git-ssh-command /usr/local/bin
       echo "git-ssh-command" >> $MINT_ENV/GIT_SSH_COMMAND
-      if [[ -n "$GITHUB_ACCESS_TOKEN" ]]; then
+      if [[ -n "$GIT_SSH_KEY" ]]; then
+        echo -n "" > $MINT_VALUES/credentials-arg
+      else
         echo "Setting credential.helper to clone using github-access-token"
         echo -n "-c credential.helper='!bash -c \"echo username=x-access-token && echo password=\${GITHUB_ACCESS_TOKEN}\"'" > $MINT_VALUES/credentials-arg
-      else
-        echo -n "" > $MINT_VALUES/credentials-arg
       fi
     env:
-      GITHUB_ACCESS_TOKEN: ${{ params.github-access-token }}
+      GIT_SSH_KEY: ${{ params.ssh-key }}
```

Comment:
This is just for more straightforward caching since SSH keys likely aren't getting rotated hourly. It'd be sweet if we had a way to configure cache only on the presence/absence of something.

Reply:
Good call